src/MessageStrip/__stories__/MessageStrip.stories.js

```diff
@@ -0,0 +1,54 @@
+import MessageStrip from '../MessageStrip';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+    boolean,
+    select,
+    text,
+    withKnobs
+} from '@storybook/addon-knobs';
+
+
+const createProps = (overrides) => ({
+    dismissible: boolean('dismissible', false),
+    noGlyph: boolean('noGlyph', false),
+    type: select('Validation State', {
+        'default': null,
+        'warning': 'warning',
+        'error': 'error',
+        'success': 'success',
+        'information': 'information'
+    }),
+    linkText: text('linkText', 'Default MessageStrip'),
+    link: text('href', ''),
+    ...overrides
+});
+
+storiesOf('Components|MessageStrip', module)
+    .addDecorator(withKnobs)
+    .add('Dev', () => (
+        <MessageStrip {...createProps()}>Default MessageStrip</MessageStrip>
+    ))
+    .add('Dismissable', () => (
+        <MessageStrip dismissable>MessageStrip</MessageStrip>
```

Comment:
I think it's spelled `dismissible`.  But this does seem like the right spelling

Reply:
oh good catch - I can't spell 🤣

Reply:
`dismissible` is the correct spelling - I looked through the history and I've had to fix this a couple times because we(me) can't get it right. Maybe we change the prop name in the future 😆